src/components/ContentHighlights/tests/DeleteHighlightSet.test.jsx

```diff
@@ -97,7 +97,7 @@ describe('<DeleteHighlightSet />', () => {
     expect(sendEnterpriseTrackEvent).toHaveBeenCalledTimes(1);
   });
 
-  it('cancelling confirmation modal closes modal', () => {
```

Comment:
Just trying to make sure we're using only the US spelling throughout the code

Reply:
I never knew the double "l" was more used outside of the US! :)

Reply:
See I have to catch myself because apparently I use british spelling for a lot of stuff accidentally? Like grey (which is gray in the US?)